react/backstop/storybook.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Determines if a component should use viewports for atoms.
+ *
+ * @param {object} component
+ * @returns {boolean}
+ */
+const isAtom = (component) => {
+  const { kind } = component;
+  // Skip table and media/Image; they need to be tested with larger viewports.
+  return(
+    kind.includes('atoms')
+    && !kind.includes('table')
+    && !kind.includes('Image')
+  );
+};
+
+/**
+ * Determines if a component is the Fixed Feedback button.
+ *
+ * @param component
+ * @return {boolean}
+ */
+const isFixedFeedback = (component) => component.kind.includes('ButtonFixedFeedback');
+
+const mapComponents = (components, debug = false) => Object.values(components)
+  .filter((component) => !['/animations', '/styles', 'ButtonCopy'].includes(component.kind))
+  .map((component) => {
+    const {
+      id, kind, name, parameters
+    } = component;
+    const viewports = [];
+    const selectors = [];
+    if (isAtom(component)) {
+      viewports.push({ label: 'small_atom', width: 400, height: 250 });
+    } else {
+      viewports.push({ label: 'phone', width: 320, height: 480 });
+      viewports.push({ label: 'tablet', width: 1024, height: 768 });
+    }
+    // Set the component selector. For the fixed feedback button we need to use
+    // an explicit selector. For all others, snapshot Storybook's render screen.
+    if (isFixedFeedback(component)) {
+      selectors.push('.ma__fixed-feedback-button');
+    } else if (parameters && parameters.docsOnly) {
+      selectors.push('#docs-root');
+    } else {
+      selectors.push('#root');
+    }
+
+    let urlBase = 'http://web/';
+    if (debug) {
+      // Only use --debug when running backstop outside of docker for local
+      // testing & debugging. Images will be generated with your host OS's browser
+      // and will not match the images generated when running in CircleCI.
+      urlBase = 'http://localhost:6006/';
+    }
+    // Backstop overrides.
+    const overrides = [
+      'GeneralTeaser',
+      'Icon',
+      'ButtonCopy'
```

Comment:
I went through the backstop report, not sure why there are no reference images, is it because the name of the screenshot files all have changed

Reply:
can you disable backstop snapshots for these as they only show an image of the error

vrt_about_Mayflower_React_Page_0_docs-root_1_tablet.png
vrt_about_Mayflower_React_Page_0_docs-root_0_phone.png
vrt_atoms_buttons_ButtonCopy_Default_0_root_0_small_atom.png
vrt_molecules_Tags_Default_0_root_1_tablet.png

Reply:
@smurrayatwork  ^